react/features/invite/components/security-dialog/index.js

```diff
@@ -0,0 +1 @@
+export { default as SecurityDialog } from './SecurityDialog';
```

Comment:
Can you please make sure you add `// @flow` to every file? Even index files, it's very important.